### PR TITLE
README: update Ecosystem

### PR DESCRIPTION
**Pull request by florianl**
Add NETLINK_SOCK_DIAG to the ecosystem.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,11 +165,19 @@ flowchart LR
         click go-onewire "https://github.com/SpComb/go-onewire"
     end
 
+    subgraph "NETLINK_SOCK_DIAG"
+        direction LR
+
+        go-diag["github.com/florianl/go-diag"]
+        click go-onewire "https://github.com/florianl/go-diag"
```
Comment (mdlayher) [inline README.md:172]:
```suggestion
        click go-diag "https://github.com/florianl/go-diag"
```

File: README.md [hunk 2]
```diff
@@ -165,11 +165,19 @@ flowchart LR
...
+    end
+
     NETLINK_CONNECTOR --> netlink
     NETLINK_CRYPTO --> netlink
     NETLINK_GENERIC --> netlink
     NETLINK_KOBJECT_UEVENT --> netlink
     NETLINK_NETFILTER --> netlink
     NETLINK_ROUTE --> netlink
     NETLINK_W1 --> netlink
+    NETLINK_SOCK_DIAG --> netlink
```
Comment (mdlayher) [inline README.md:182]: Move just above W1 to alphabetize, thanks!

Comment (florianl) [reply README.md:182]: Sorted alphabetically :white_check_mark:

File: README.md (hunk 1 above)
Comment (florianl) [reply README.md:172]: not sure, how I missed this :see_no_evil:

**Review by mdlayher — APPROVED** [review]

**Pull request merged**